### ISSUE TITLE
Included + super

### ISSUE THREAD

**Issue by jscheid** [opal/opal#1900]
```ruby
class A < Module
  def included(_descendant)
    STDOUT.puts 'A included'
  end
end

class B < A
  def included(_descendant)
    super
    STDOUT.puts 'B included'
  end
end

class C
  include B.new
end
```

ruby 2.5.1p57 prints:
```
A included
B included
```

Opal 0.11.4 prints:
```
Error: could not find current class for super()
    at Object.Opal.find_obj_super_dispatcher ([stdin]:1293:13)
    at Object.Opal.find_super_dispatcher ([stdin]:1254:25)
    at module_constructor.$$included [as $included] ([stdin]:21654:24)
    at singleton_class_alloc.$$include [as $include] ([stdin]:3019:15)
    at [stdin]:21664:17
    at [stdin]:21665:5
    at [stdin]:21666:3
    at Script.runInThisContext (vm.js:96:20)
    at Object.runInThisContext (vm.js:303:38)
    at Object.<anonymous> ([stdin]-wrapper:6:22)
```

**Comment by elia** [comment]
**I've just opened #1901 that makes `master` work properly**. The module management and the super dispatching in 0.11 is very complex and easily broken in corner cases (as you saw). For that reason **I don't think I'll even try to fix 0.11.x** unless there's a very good reason too, if there. 

Let me know if there a good reason otherwise and I'll reconsider 👍, if not try switching to `master` instead.

**Comment by jscheid** [comment]
Perfect, thanks for the quick fix! I don't need it in 0.11.
